sdk/identity/identity/package.json

```diff
@@ -84,10 +87,24 @@
     "cross-env": "^5.2.0",
     "eslint": "^5.16.0",
     "inherits": "^2.0.3",
+    "karma": "^4.0.1",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage": "^1.1.2",
+    "karma-edge-launcher": "^0.4.2",
```

Comment:
We're currently only using chrome headless. Are these dependencies needed? I'm guessing it's also due to the copy & paste, but I'm genuinely curious since I saw that a week ago and I couldn't figure whether we were running tests against more browsers or not 🤔

Reply:
Another case of copy/paste.  It'd be nice to test on other browsers too but we might need a shared standardized Karma config to do that well.  I'll remove the unnecessary deps for now.

Reply:
Uhhh I like the words _shared_ and _standardized_. Let me know if we should open an issue about it!

Reply:
Instead of removing the launchers, we should make it a priority to extend the testing to cover more browsers. :)

Recent experiences with IE -

We didn't test the IE browser until the last week of release for storage packages.
As a consequence, we had to do debug and do last minute changes to the packages to fix a couple of issues with the browser that went unnoticed for some time.

1. https://github.com/Azure/azure-sdk-for-js/pull/4156 - Had to use an older version of a package to support older browsers. 
It'd be scary when the dependency of a package doesn't support a browser when the package does.
2. https://github.com/Azure/azure-sdk-for-js/pull/4164/commits/2ffc4c46924acc3b0edcfbea4361acc2189c4798 - Disabled a check in some test due to the difference in behavior for IE browser